init-client.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/sh
+#TODO.md
+
+# Always check if a clean install can be updated
+yum -y update
+
+if [ "$1" = "-v" ]; then
+  ANSIBLE_VERSION="${2}"
+fi
+
+yum_makecache_retry() {
```

Comment:
Does this need to be repeated in both scripts?

Reply:
Well, it could be changed to one script by using simple parameters actually :-)